src/main/java/carracing/ui/ResultView.java

```diff
@@ -21,26 +29,43 @@ public static ResultView getInstance() {
   }
 
   public void printResult(RacingResult racingResult) {
-    sb = new StringBuilder();
-    for (RoundResult result : racingResult.getResults()) {
+    stringBuilder = new StringBuilder();
+    composeResult(racingResult);
+    System.out.println(stringBuilder);
+  }
+
+  private void composeResult(RacingResult racingResult) {
+    RoundResult roundResult = racingResult.getResult();
+    appendRacingData(roundResult);
+    appendRacingWinner(racingResult);
+  }
+
+  private void appendRacingData(RoundResult roundResult) {
+    for (RoundData result : roundResult.getRoundDataList()) {
       combineRoundResult(result);
-      sb.append(System.lineSeparator());
+      stringBuilder.append(System.lineSeparator());
     }
-    System.out.println(sb);
   }
 
-  private void combineRoundResult(RoundResult result) {
-    for (RacingData racingData : result.getRacingDataList()) {
+  private void combineRoundResult(RoundData result) {
+    result.getRacingDataList().forEach(racingData -> {
       convertCarStatusToString(racingData);
-      sb.append(System.lineSeparator());
-    }
+      stringBuilder.append(System.lineSeparator());
+    });
   }
 
   private void convertCarStatusToString(RacingData racingData) {
+    stringBuilder.append(racingData.getName()).append(BLANK_STRING).append(COLON).append(BLANK_STRING);
```

Comment:
```suggestion
    stringBuilder.append(racingData.getName())
            .append(BLANK_STRING)
            .append(COLON)
            .append(BLANK_STRING);
```

적절한 개행은 코드의 가독성을 높혀줄 수 있습니다.